xwiki-rendering-macros/xwiki-rendering-macro-toc/src/main/java/org/xwiki/rendering/macro/toc/TocMacroParameters.java

```diff
@@ -46,6 +46,22 @@
          */
         PAGE
     }
+    
+    /**
+     * @version $Id$
+     */
```

Comment:
Don't forget to add a `@since 10.0RC1` in the javadoc for thi enum.

Reply:
And some javadoc :)